views/content/contact.njk

```diff
@@ -0,0 +1,22 @@
+{% extends "base.njk" %}
+
+{% block main_content %}
+  <div class="usa-grid info-block usa-content">
+    <h1>Contact</h1>
+    <h2>Want to buy Federalist?</h2>
+    <p>If you’re interested in using Federalist,
+      <a href="mailto:cloud-gov-inquiries@gsa.gov?body=What%27s%20your%20name%3F%0A%0AWhat%20agency%20or%20office%20do%20you%20work%20for%3F%0A%0AWhat%27s%20your%20job%20title%20or%20role%3F%0A%0ATell%20us%20a%20little%20about%20your%20project%20or%20your%20questions%20about%20cloud.gov:%0A%0AIf%20you%27d%20like%20us%20to%20call%20you%2C%20what%27s%20your%20phone%20number%20and%20when%20might%20be%20a%20good%20time%3F%0A">email the Federalist team</a> with your name, agency or office, and information about the website you want to build or migrate. We’ll schedule a call to answer your questions and help you get started.</p>
```

Comment:
Needs new content

Reply:
Content for the mailto: link or content for this entire page.
I pulled the latest from https://docs.google.com/document/d/1PYVw2IgRCuatdtpn1_0nKBL8JBV4i5VbyARnFJuzk3o/edit#heading=h.92izs1cnqim7